<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QDialog" name="OptionsWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2048 - Options</string>
  </property>
  <widget class="QPushButton" name="backToMenu">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Board size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="boardSize">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4x4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End goal:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="endGoal">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="goalLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light SemiCondensed</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>boardSize</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>backToMenu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
